source/features/clean-conversation-headers.tsx

```diff
@@ -15,49 +15,30 @@ async function initIssue(): Promise<void> {
 		return;
 	}
 
-	byline.classList.add('rgh-clean-conversation-header');
-
-	// Removes: [octocat opened this issue on 1 Jan] · 1 comments
-	for (let i = 0; i < 4; i++) {
-		byline.firstChild!.remove();
-	}
+	byline.classList.add('rgh-clean-conversation-header', 'rgh-clean-conversation-headers-hide-author');
```

Comment:
Why two classes? One should be enough

Reply:
Because the `-hide-author` class is reused. It should always apply to issues but only for PRs with same author and meger.